articles/tokens/refresh-token.md

```diff
@@ -1,152 +1,173 @@
+    "queryString" : [],
+    "postData" : {
+      "mimeType": "application/json",
+      "text" : "{ \"client_id\": \"${account.clientId}\", \"client_secret\": \"${account.clientSecret}\", \"token\": \"YOUR_REFRESH_TOKEN\" }"
+    },
+    "headersSize" : 150,
+    "bodySize" : 0,
+    "comment" : ""
 }
 ```
 
-Response body:
-
-```json
-[
-  {
-    "id": "dcr_dFJiaAxbEroQ5xxx",
-    "device_name": "my-device" // the value of 'device' provided in the /authorize call when creating the token
-  }
-]
-```
+Where:
+- `client_id`: Your application's Client ID.
+- `client_secret`: Your application's Client Secret.
+- `token`: The refresh token you want to revoke.
 
-To revoke a __refresh token__, call the [Delete a device credential](/api/management/v2#!/Device_Credentials/delete_device_credentials_by_id) endpoint with an access token containing `delete:device_credentials` scope and the value of `id` obtained above:
+The client should match the one the refresh token was issued for.
 
-```text
-DELETE https://${account.namespace}/api/v2/device-credentials/{id}
+If the request is valid, the refresh token is revoked and the response is `HTTP 200`, with an empty response body. Otherwise, the response body contains the error code and description.
 
+```json
 {
-  "Authorization":   "Bearer {your_access_token}"
+  "error": "invalid_request|invalid_client",
+  "error_description": "Description of the error"
 }
-
 ```
 
-The response will be a **204**: The credential no longer exists.
+The possible responses are:
+
+| HTTP Status | Description |
+| --- | --- |
+| 200 | The refresh token is revoked or does not exist. The response body is empty. |
+| 400 | The required parameters were not sent in the request or the refresh token was not issued to the client making the revocation request (`"error": "invalid_request"`). |
+| 401 | The request is not authorized (`"error": "invalid_client"`). Check that the client credentials (`client_id` and `client_secret`) are present in the request and hold valid values. |
 
-### Revoke a Refresh Token in the Dashboard
+### Revoke a refresh token using the dashboard
 
-To see if a user has existing devices with associated refresh tokens, go to the [Users section](${manage_url}/#/users) of the dashboard. Click the name of the user to view their **Details** page.
+To revoke a refresh token, go to the [Users section](${manage_url}/#/users) of the [dashboard](${manage_url}). Click the name of the user to view their *Details* page.
 
-Select the **Devices** tab. This page lists all device names and the number of refresh tokens associated with each. To revoke a refresh token, click the **X** to the right of the device name.
+Select the *Authorized Applications* tab. This page lists all the clients to which the user has authorized access. Revoking an authorized application revokes also its associated refresh tokens.
 
-![](/media/articles/tokens/dashboard-revoke-refresh-token.png)
+To revoke a refresh token, click **Revoke**.
```

Comment:
Subtle difference, we don't revoke the token in the dashboard. We delete the underlaying `user grant`. This is grant is used to validate that a refresh token should get a new access token / id token for that particular app. If the grant gets revoked, then the refresh token becomes useless.

Maybe we can switch the writing here to talk about "Revoking access to an Authorized Application will cause any refresh token issue to that application to become invalid."